tests/cover/test_regressions.py

```diff
@@ -55,3 +55,23 @@ def test(v):
 
     with pytest.raises(AssertionError):
         test()
+
```

Comment:
Here's the regression test for 491 - it does fix the bug reported in 491, so I updated the notes accordingly.